### PR TITLE
Update model and style

### PR DESCRIPTION
**Pull request by emacha**
Updated the model for a 90% cheaper one. Also took the DBT guide and trimmed it down to see how it works with real guides.

I ran on one of our DBT repo queries. _almost_ commited it before I realised it was outside Zego....

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -27,23 +28,23 @@
...
 
-response = openai.Completion.create(
-  model="text-davinci-003",
-  prompt=prompt,
+response = openai.ChatCompletion.create(
+  model="gpt-3.5-turbo",
+  messages=[
+        {"role": "system", "content": system_prompt},
+        {"role": "user", "content": sql_file}
+    ],
```
Comment (SentientHousePlant) [inline main.py:41]: Beautifully done!

File: sql_style.md [hunk 2]
```diff
@@ -1,5 +1,144 @@
...
+
+- Fields should be stated before aggregates / window functions
+
+- Aggregations should be executed as early as possible before joining to another table.
+
+- Ordering and grouping by a number (eg. group by 1, 2) is preferred over listing the column names (see [this rant](https://blog.getdbt.com/write-better-sql-a-defense-of-group-by-1/) for why). Note that if you are grouping by more than a few columns, it may be worth revisiting your model design. If you really need to, the [dbt_utils.group_by](https://github.com/dbt-labs/dbt-utils/tree/0.8.6/macros/sql/groupby.sql) function may come in handy.
+
+- Prefer `union all` to `union` [*](http://docs.aws.amazon.com/redshift/latest/dg/c_example_unionall_query.html)
+
+- Avoid table aliases in join conditions (especially initialisms) – it's harder to understand what the table called "c" is compared to "customers".
```
Comment (SentientHousePlant) [inline sql_style.md:140]: The example should really help :) I can't wait to see how it performs with this style guide!

File: sql_style.md [hunk 3]
```diff
@@ -1,5 +1,144 @@
...
+
+## Misc
+
+- Don't misgender people!
```
Comment (SentientHousePlant) [inline sql_style.md:144]: 🤘🏳️‍⚧️

**Review by SentientHousePlant — APPROVED** [review]
My name is Commander Shepard and this is my favourite PR.

File: main.py (hunk 1 above)
Comment (emacha) [reply main.py:41]: That was half GPT-3 tbh. I switched the model then it started erroring out. I was only copy pasting stack traces to chatgpt then copying the reponse back

**Pull request merged**